packages/react/runtime/src/index.ts

```diff
@@ -33,18 +33,13 @@ import {
   useRef,
   useState,
 } from './hooks/react.js';
-import { loadLazyBundle, mainThreadLazy } from './lynx/lazy-bundle.js';
+import { loadLazyBundle } from './lynx/lazy-bundle.js';
 
 export { Component, createContext } from 'preact';
 export { PureComponent } from 'preact/compat';
 export * from './hooks/react.js';
 
-const lazy: typeof import('preact/compat').lazy = /*#__PURE__*/ (() => {
-  lynx.loadLazyBundle = loadLazyBundle;
-  return __LEPUS__
-    ? mainThreadLazy
-    : backgroundLazy;
-})();
+lynx.loadLazyBundle = loadLazyBundle;
```

Comment:
We should keep the IIFE here to save some bytes when `lazy` is not used.

```suggestion
const lazy: typeof import('preact/compat').lazy = /*#__PURE__*/ (() => {
  lynx.loadLazyBundle = loadLazyBundle;
  return preactLazy;
});
```

This would save like 1 kB for the hello world project :)